app/src/main/java/com/example/weather/ui/temp/TempViewModel.java

```diff
@@ -55,11 +62,53 @@ public Completable deleteWeatherById(int id){
     @SuppressLint("CheckResult")
     public void fetchDailyWeather(int id) {
         getWeatherById(id).subscribe(weatherData -> {
-            getDailyWeatherByCoord(weatherData.getLan(), weatherData.getLon())
+            getDailyWeatherByCoord(weatherData.getLan(), weatherData.getLon(),0)
+                    .subscribe(hashMap -> {
+                        Map<String, String> tmp = liveData.getValue();
+                        tmp.put(Constant.MORN, hashMap.get(Constant.MORN_MAPPER));
```

Comment:
I'd recommend to change this. It isn't scalable. What if we are going to display more weathers? Will we add more constants? I'd recommend to make a list.